admin/solidus_admin.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'geared_pagination', '~> 1.1'
   s.add_dependency 'importmap-rails', '~> 1.2', '>= 1.2.1'
   s.add_dependency 'solidus_backend'
+  s.add_dependency 'solidus_legacy_promotions'
```

Comment:
Would be great if this would not be necessary, but I see why we need this for now.